sdk/keyvault/Azure.Security.KeyVault.Administration/tests/BackupRestoreClientLiveTests.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System;
+using System.Threading;
+using Azure.Security.KeyVault.Administration.Models;
+
+namespace Azure.Security.KeyVault.Administration.Tests
+{
+    public class BackupRestoreClientLiveTests : BackupRestoreTestBase
+    {
+        public BackupRestoreClientLiveTests(bool isAsync) : base(isAsync, RecordedTestMode.Record /* To record tests, change this argument to RecordedTestMode.Record */)
```

Comment:
You should change this to `Playback`.

Reply:
You might also want to add a comment to get the right connection information from the service team (don't document it publicly).